qiskit/providers/__init__.py

```diff
@@ -115,14 +116,14 @@
 
 If you have a quantum device or simulator that you would like to integrate with
 Qiskit you will need to write a provider. A provider will provide Terra with a
-method to get available :class:`~qiskit.provider.BackendV1` objects. The
-:class:`~qiskit.provider.BackendV1` object provides both information describing
+method to get available :class:`~qiskit.providers.BackendV1` objects. The
+:class:`~qiskit.providers.BackendV1` object provides both information describing
 a backend and its operation for the :mod:`~qiskit.transpiler` so that circuits
 can be compiled to something that is optimized and can execute on the
 backend. It also provides the :meth:`~qiskit.providers.BackendV1.run` method which can
 run the :class:`~qiskit.circuit.QuantumCircuit` objects and/or
 :class:`~qiskit.pulse.Schedule` objects. This enables users and other Qiskit
-APIs, such as :func:`~qiskit.execute.execute` and higher level algorithms in
+APIs, such as :func:`~qiskit.execute_function.execute` and higher level algorithms in
```

Comment:
```suggestion
APIs, such as :func:`~qiskit.execute` and higher level algorithms in
```

Reply:
What's in the PR is actually correct. It needs to be the full path for sphinx to resolve it correctly

Reply:
This doesn't actually affect how it appears (the tilde `~` truncates it to the last word anyway), it's just for defining how Sphinx should find the symbol.  It looks things up in their canonical representation, so `qiskit.execute` doesn't resolve to a cross-ref-able symbol, unfortunately (at least in my local docs build).  `` :obj:`.execute` `` _might_ resolve correctly, but I wouldn't want to bet on it always resolving correctly with a generic name.

Sphinx works in very mysterious ways :(

Reply:
I was not aware of this. Great to learn it :)